.github/workflows/pylint.yml

```diff
@@ -2,26 +2,24 @@ name: Pylint
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python 3.9
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.9
-    - name: Install Dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-    - name: Analysing the code with pylint
-      run: |
-        pylint bookwyrm/
-
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
```

Comment:
We should change this line to match the version, since it now sets up Python 3.11.

Reply:
The problem I ran into last time is not matching what upstream does. I don't know that I feel confident taking on the whole of upgrading a python version for this repo? Thoughts so, so welcome.

Reply:
So, the line after this actually sets the version, and it's already on 3.11 here, so this comment was more about updating the name to match what it does.

Upstream has changed everything significant that needs to change to support this upgrade already, though (that was part of the huge batch of changes we took recently). I think this would be fine for us at this point, and if we get have it working well in our prod that's a great argument to commit a similar upgrade to upstream.